code/modules/shuttle/on_move.dm

```diff
@@ -13,8 +13,8 @@ All ShuttleMove procs go here
 // Called from the new turf before anything has been moved
 // Only gets called if fromShuttleMove returns true first
 /turf/proc/toShuttleMove(turf/oldT, shuttle_dir)
-	for(var/i in 1 to contents.len)
-		var/atom/movable/thing = contents[i]
+	for(var/i in contents)
+		var/atom/movable/thing = i
```

Comment:
remie specifically said that the `in 1 to len` loop is faster than the `in list` and had me change a whole bunch of them for that reason